pkg/larker/testdata/load-git-helpers/.cirrus.star

```diff
@@ -1,5 +1,9 @@
 load("github.com/cirrus-templates/helpers@a5e5d1649c05c40bab6c82f084b69a8d82977d96", "task", "container", "script", "always", "artifacts")
 
+# Ensure that we can load by hash and a branch name
+load("github.com/cirrus-templates/graphql@1f72f65b1d0aaa6052f2401cad2701e2bd3bd12d", "rerun_task_from_hash")
+load("github.com/cirrus-templates/graphql@initial-queries", "rerun_task_from_branch")
```

Comment:
This branch will probably go away. Maybe create a tag/release in `github.com/cirrus-templates/helpers` and add it?

Reply:
It needs to be a branch.

I've created one [here](https://github.com/cirrus-templates/helpers/tree/branch-for-cli-testing) and refactored the code again to fetch all refs, otherwise it is difficult to checkout an arbitrary hash, branch or tag, see https://github.com/cirruslabs/cirrus-cli/pull/367/commits/32e828f0afc100ba2f8bc8f9d6a2048f4b8a155b.